### ISSUE TITLE
codecov needs token now

### ISSUE THREAD

**Issue by JeffFessler** [JuliaLinearAlgebra/LinearMaps.jl#228]
See
https://discourse.julialang.org/t/psa-new-version-of-codecov-action-requires-additional-setup/109857

**Comment by dkarrasch** [comment]
Thanks for the notice. In that thread, I read "organizational tokens". Does that mean we can solve this at the organizational (i.e., JuliaLinearAlgebra) level? I'll try to take a look after the weekend. Good that there is not much development at the moment. :see_no_evil:

**Comment by JeffFessler** [comment]
> Does that mean we can solve this at the organizational (i.e., JuliaLinearAlgebra) level?

Yes, I think so, but I'm unsure.  For years I have looked at your code and workflows for help and I was hoping you'd have this figured out already so I could again copy you :)

**Comment by fredrikekre** [comment]
There is already an organizational token added to this org. Is it not working?

**Comment by JeffFessler** [comment]
> There is already an organizational token

That's great!  Then hopefully all that is needed is the new token line in yml file.
